.github/workflows/spec.yml

```diff
@@ -0,0 +1,29 @@
+name: Specification
+
+on:
+  pull_request:
```

Comment:
Can you also run on pushes to main?

https://github.com/commonwarexyz/monorepo/blob/54b504ff2be93f1897952932da43c40950a01c6c/.github/workflows/fast.yml#L4-L5